k8s-readonly.tf

```diff
@@ -34,14 +34,19 @@ resource "kubernetes_deployment" "apiary_hms_readonly" {
         }
         annotations = {
           "iam.amazonaws.com/role" = aws_iam_role.apiary_hms_readonly.name
+          "prometheus.io/path"     = "/metrics"
+          "prometheus.io/port"     = "8080"
+          "prometheus.io/scrape"   = "true"
         }
       }
 
       spec {
         container {
           image = "${var.hms_docker_image}:${var.hms_docker_version}"
           name  = "hms-readonly"
-
+          port {
+            container_port = 9083
```

Comment:
What port is this and is it different to what it was before? (if so don't we need to document this in the CHANGELOG?)

Reply:
@massdosage - was wondering the same thing, so I did some digging in.  The k8s service definition already defined the thrift port `9083` as the port.  Adding it to the k8s deployment seems to be informational only, in the same vein as the docker `EXPOSE` directive.  https://medium.com/faun/should-i-configure-the-ports-in-kubernetes-deployment-c6b3817e495

Reply:
Added this as prometheus service disovery looks this port, then replaces with exporter port when exporter is running on different port.

Reply:
oh, interesting...  Thank you for the explanation.

Reply:
Thanks for the info.